application/backend/src/main/java/pl/ds/bulma/components/models/BreadcrumbElementComponent.java

```diff
@@ -42,4 +43,11 @@ public class BreadcrumbElementComponent {
   @Getter
   @Default(values = StringUtils.EMPTY)
   private String icon;
+
+  @SlingObject
+  private Resource resource;
```

Comment:
If you need only a ResourceResolver then you don't have to use a Resource.  This should works as well:

```
  @SlingObject
  private ResourceResolver resourceResolver;
```